### PR TITLE
Randomize beneficiary listing

### PR DESCRIPTION
**Pull request by xvd3**

### CODE REVIEW

File: services/beneficiaries/random.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+function service(data, is_admin) {
+  const d = q.defer();
+  let params = {};
+
+  q.fcall(async () => {
+    const result = morx.validate(data, spec, { throwError: true });
+    params = result.params;
+    params.count = params.count || DEFAULT_COUNT;
+    params.count = params.count * 1;
```
Comment (oreHGA) [inline services/beneficiaries/random.js:19]: Curious, any reason why we're multiplying by 1?

Comment (xvd3) [reply services/beneficiaries/random.js:19]: @oreHGA That bit's to (implicitly) type-case params.count to 1.
Not assuming that the client will always pass a valid number. 
Doing * 1, ensures that if we have cases of /randomize-people?count=zzzzzzz the zzzz * 1 will cause isNaN to be true, forcing the code to default count to 10

Comment (oreHGA) [reply services/beneficiaries/random.js:19]: Ah nice!

**Comment by oreHGA** [comment]
Looks good to merge!

**Comment by xvd3** [comment]
@oreHGA Fields added.

**Comment by oreHGA** [comment]
Closing this as changes are now here :-> #9

**Pull request closed without merging**